agent/containers/images/Makefile

```diff
@@ -39,9 +39,9 @@ _WORKLOAD_RPMS = fio uperf
 _ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
 
 # By default we only build images for the following distributions:
-_DISTROS = centos-8 centos-7 fedora-32 fedora-31
+_DISTROS = centos-8 centos-7 fedora-33 fedora-32
```

Comment:
Hmm. Not really a big deal here since building container images for the agent is recent and not widely distributed; but in general the idea of changing the support matrix for an old release (0.69) seems ... odd.

Reply:
We have not choice in the matter.  COPR no longer allows us to build Fedora 31 RPMs.  It is end-of-life.  We can setup our own build system for past versions, but that is well beyond the scope of this PR.